recipe-server/normandy/settings.py

```diff
@@ -162,7 +164,15 @@ def CSP_DEFAULT_SRC(self):
     CSRF_COOKIE_NAME = 'csrftoken-20170707'
 
 
-class Base(Core):
+class CORS:
```

Comment:
:+1: I like having a mixin for this kind of setting. It gives a nice bit of organization.

Reply:
I contemplated calling it `CORSMixin` but thought it was a bit overkill considering that this isn't much object oritentation, but more a classes-as-groupings.